### DISCUSSION TITLE
报数声音

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#39] [想法]
在 销售窗口 进行报数声音

**Comment by densen2014** [comment]
老板说点中文，我又不是神仙，不知道你在说啥

**Comment by aaronameixa** [comment]
在算账时候，每一次在刷一个产品进行一次报数 （数量）
![image](https://github.com/densen2014/docs/assets/122285019/b838dd92-68c0-48ce-8513-9debf563237d)

**Reply by densen2014** [reply]
你联系客服登记一下吧，我不太看得懂你这个需求，或者说详细的操作步骤，我不是很懂

**Comment by densen2014** [comment]
V7.12.7

数量改变就会播报, 输入新行或者手动改变包数都会激活播报功能


视频

https://youtu.be/I2Leux1RMIQ


![image](https://github.com/densen2014/docs/assets/8428709/018bbc5c-6729-4d52-9bb7-d26bd0204761)

**Reply by aaronameixa** [reply]
你这个版本那里下载？

**Reply by densen2014** [reply]
你看上面，改变包的数量才会报数字。应该是原始就这样设计的。

**Reply by aaronameixa** [reply]
我试过在最新版本， 报的不是数量 ， 而是金额

**Comment by densen2014** [comment]
![image](https://github.com/densen2014/docs/assets/8428709/006a833a-bf28-45a9-84e6-a08a3ad05e79)

**Comment by densen2014** [comment]
v8.9.2-beta修复

**Reply by aaronameixa** [reply]
这个对应着 240919-beta？

**Reply by densen2014** [reply]
反正装最新的就行，具体不知道是哪一天了
